src/screens/FeedScreen/Header.js

```diff
@@ -28,7 +36,11 @@ import useFeedHeader from './hooks/useFeedHeader';
 import {DEFAULT_PROFILE_PIC_PATH, PRIVACY_PUBLIC} from '../../utils/constants';
 import {calculateTime} from '../../utils/time';
 import {colors} from '../../utils/colors';
-import {fonts} from '../../utils/fonts';
+import {fonts, normalizeFontSize} from '../../utils/fonts';
+import ShareAndroidIcon from '../../assets/icons/images/share-for-android.svg';
+import TrashRed from '../../assets/icons/images/trash-red.svg';
+import BottomSheetMenu from '../../components/BottomSheet/BottomSheetMenu';
+import ShareUtils from '../../utils/share';
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

New imports have been added to support the new features introduced in this PR. These include a new SVG icon for sharing on Android, a trash icon for deleting posts, and a utility for sharing functionality.
```diff
+ import {fonts, normalizeFontSize} from '../../utils/fonts';
+ import ShareAndroidIcon from '../../assets/icons/images/share-for-android.svg';
+ import TrashRed from '../../assets/icons/images/trash-red.svg';
+ import BottomSheetMenu from '../../components/BottomSheet/BottomSheetMenu';
+ import ShareUtils from '../../utils/share';
```


<!-- This is an auto-generated comment by OSS CodeRabbit -->